package/yast2-dhcp-server.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Jan 24 10:52:56 CET 2017 - schubi@suse.de
+
+- Fixed a crash at start when the "dhcp-server" package was not
+  installed in the system. The crash happens with the latest
+  yast2-core and yast2-ruby-bindings packages (it is related to the
```

Comment:
Please open a separate bug for it, this description is actually only partly true. Originally I thought this was not a problem in SP2 and it was revealed by updated core and ruby-bindings. Which is not true as you tested.

To make the changes entry better understandable for customers we need a new bug report and remove that reference to new core and ruby-bindings from the changes.